progress.go

```diff
@@ -32,6 +33,9 @@ type Progress struct {
 	BinlogStreamerLag       float64 // seconds
 	Throttled               bool
 
+	// The number of data iterators currently active.
+	ActiveDataIterators int
```

Comment:
Would be nice to show especially this on in the UI.

Reply:
Yeah I think a ControlServer refactor is coming after #227 cc @Manan007224